config.xml

```diff
@@ -29,8 +29,8 @@
         <icon density="xxhdpi" src="res/android/xxhdpi/icon.png" />
         <icon density="xxxhdpi" src="res/android/xxxhdpi/icon.png" />
     </platform>
-    <engine name="android" spec="^6.3.0" />
     <plugin name="cordova-plugin-whitelist" spec="^1.3.3" />
     <plugin name="acidhax.cordova.chromecast" spec="https://github.com/jellyfin/cordova-plugin-chromecast" />
     <plugin name="org.jellyfin.mobile" spec="./src/NativeShell" />
+    <engine name="android" spec="^8.0.0" />
```

Comment:
does this mean Android older than Oreo is doomed?

Reply:
No, thats only a indicator for the cordova-android version being used. If you go to package.json you will see that the package cordova-android is targeted to at least 8.0.0